karpenter/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.0 / 2023-11-10
+
+***Added***:
+
+* Initial release ([#16082](https://github.com/DataDog/integrations-core/pull/16082))
+
```

Comment:
Well, looks like it's not correctly formatted

Reply:
Where do you see the problem?

Reply:
```
# CHANGELOG - karpenter

<!-- towncrier release notes start -->
```

is after 

```
## 1.0.0 / 2023-11-10

***Added***:

* Initial release ([#16082](https://github.com/DataDog/integrations-core/pull/16082))
```

Reply:
I think you should manually fix it and create a card to investigate

Reply:
Uh, karpenter changelog is missing newline after the `<!-- towncrier release notes start -->`. We need the newline to match according to our config.

Reply:
I'll fix this manually here, but I'll update the jira card we have for the new integration template's changelog to include this.